2017/JULY/JULY17/CALC/CALC.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+	Made by:
+		``` Monal Shadi ``
+	Github: Monal5031
+	Codechef: monalshadi
+*/
+
+#include "bits/stdc++.h"
+
+using namespace std;
+
+int main() {
+	ios::sync_with_stdio(false)
+	cin.tie(NULL);
+	cout.tie(NULL)
+
+	int tc;
+	cin >> tc;
+	while (tc--) {
+		long long n, b;
+		cin >> n >> b;
+		if (n <= b) {
+			cout << "0\n";
+			continue;
+		}
+		else if (b > n / 2) {
+			cout << n % b << "\n";
+			continue;
+		}
+		unsigned long long a = n / b;
+		unsigned long long t = a;
+		a = ceil(a / 2.0);
```

Comment:
`a = a / 2 + a % 2;` could be better

Reply:
Actually we are not supposed to change the code where it may change the time of execution which was in actual submission. views? @aashutoshrathi @Lucifer24

Reply:
Exactly, we shouldn't make changes that actually change run time of the code.